docs/guides/docs/intro-to-sway/checkpoint.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Checkpoint
+category: Intro to Sway
+parent:
+  label: Guides
+  link: /guides
+---
+
+## Sway Contract Checkpoint
+
+If you have followed the previous steps correctly your `main.sw` marketplace contract should look like this
+
+<CodeImport
+  file="../../examples/intro-to-sway/contract/src/main.sw"
+  comment="all"
+  commentType="//"
+  lang="sway"
+/>
+
+## Building the contract
+
+Here's a polished version of your instructions:
+
+To format your contract, execute the command:
+```sh
+forc fmt
+```
+
+To compile your contract, navigate to the contract folder and run:
+```sh
+forc build
+```
+Congratulations! You've successfully written a full contract in Sway!
+
+Post-compilation, the system will automatically generate `abi.json`, `storage_slots.json`, and `contract.bin`. You can locate these files in the following directory:
+```
+contract/out/debug/*
+```
+
+## Deploying the contract
+
+For detailed steps on deploying this contract, refer to the official Fuel developer quickstart guide: 
+[Deploy the Contract](https://docs.fuel.network/guides/quickstart/building-a-smart-contract/#deploy-the-contract)
```

Comment:
Should we use relative links instead of DNS? 🤔

Reply:
Good call on this